docker-compose-generator/docker-fragments/opt-add-zmq.yml

```diff
@@ -0,0 +1,38 @@
+version: "3"
+
+services:
+  bitcoind:
+    environment:
+      BITCOIN_EXTRA_ARGS: |
+        zmqpubrawblock=tcp://0.0.0.0:28332
```

Comment:
For `teos` we also need `zmqpubhashblock=tcp://0.0.0.0:28332`

Reply:
We can't do this, we have `zmqpubrawtx` mapped to a different port that other services already use

Reply:
`zmqpubhashblock` you mean?

If so, teos only needs `zmqpubhashblock`, so mapping it to that one would do.